byterun/fiber.c

```diff
@@ -280,9 +254,11 @@ void caml_scan_stack(scanning_action f, void* fdata, struct stack_info* stack)
 
 struct stack_info* caml_switch_stack(struct stack_info* stk)
```

Comment:
I guess this can be inlined in `amd64.S` now. No need to hop through C for switching OCaml stacks.

Reply:
Yep, that sounds good. Still need to jump out to C for freeing the old stack, and native code has a DWARF-related hack in the return value that need to be replaced before this gets inlined.